### PR TITLE
Rework of PR #2

### PR DESCRIPTION
**Pull request by sja**
Changed the same things, but with suggestions of others like bump to ESH 0.9.0.b4.

Signed-off-by: Sebastian Janzen <sebastian.janzen@innoq.com>

### CODE REVIEW

**Comment by sja** [comment]
Sorry for the flapping of force pushes, new GPG setup ...

**Comment by kaikreuzer** [comment]
Thanks @sja!
@jochenhiller: Would you have the chance to briefly review, if this is what you would have done, if you had found the time? :-)

**Comment by JochenHiller** [comment]
Thanks @sja too
I will do the review next minutes

File: distro/runtime/concierge/smarthome.xargs [hunk 1]
```diff
@@ -23,6 +23,7 @@
 # -Dorg.eclipse.concierge.debug.packages=true
 # -Dorg.eclipse.concierge.debug.services=true
 # -Dorg.eclipse.concierge.debug.classloading=true
+-Dorg.eclipse.concierge.debug.resolver=true
```
Comment (JochenHiller) [inline distro/runtime/concierge/smarthome.xargs]: I would comment it out: it was just missing as an option which could be activated
By default it should be NOT active
So: 
`# -Dorg.eclipse.concierge.debug.resolver=true`

File: distro/runtime/concierge/start.sh [hunk 2]
```diff
@@ -1,51 +1,197 @@
...
+    echo "JAVA_VENDOR=$JAVA_VENDOR"
+}
+
+checkJvmVersion() {
+    # Minimal requirement: Java8 with compact2 profile
+    if [ "$JAVA_VERSION" -lt "80" ]; then
+        echo "ERROR: JVM must be greater than 1.7"
+        exit 2;
+    fi
+    if [ "$JAVA_COMPACT_PROFILE" = "compact1" ]; then
```
Comment (JochenHiller) [inline distro/runtime/concierge/start.sh]: As pointed out in previous comments: up to now due to logback and Jersey a Full-JRE is required.
So we have to check here for fulljre until this has been solved
`if [ "$JAVA_COMPACT_PROFILE" != "fulljre" ]; then`

File: distro/runtime/concierge/start.sh [hunk 3]
```diff
@@ -1,51 +1,197 @@
...
+        echo "ERROR: JVM must be at least compact2 profile"
```
Comment (JochenHiller) [inline distro/runtime/concierge/start.sh]: JVM must be at least Full-JRE up to now

**Review by JochenHiller — COMMENTED** [review]
Looks good, all changes are in and almost all comments from Kai are considered.
Just the debug option for resolver has to be commented, and the check changed against fulljre JVM
Thanks for doing my work

**Comment by sja** [comment]
Added your suggestions/improvements.

> Thanks for doing my work

Well, thanks for this repo! :-)

**Review by JochenHiller — APPROVED** [review]

**Comment by kaikreuzer** [comment]
Many thanks to both of you!

**Pull request merged**